src/user-management/user-management.ts

```diff
@@ -544,4 +544,8 @@ export class UserManagement {
 
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
+
+  getJwksUrl(clientId: string): string {
+    return `${this.workos.baseURL}/sso/jwks/${clientId}`;
```

Comment:
I think we should add some checks in here to make sure users don't trip up. Let's check that `clientId` isn't an empty string and that it starts with `client_`. Let's also add tests for these edge cases.

Reply:
We can't check for `client_` because not all clients start with that (lots start with `project_`, for example (i'm not sure if there are other variations). i'll add a check for empty strings though.